### PR TITLE
1536 10.3.13 release notes

### PR DESCRIPTION
**Pull request by kirstiaa**
Minor changes for the 10.3.13 release.

### CODE REVIEW

File: release-notes/10.3/index.md [hunk 1]
```diff
@@ -108,7 +111,12 @@ We use **pilot programs** to collect feedback, identify potential problems, and
 
 #### New user guides
 
-Check out the new user guide for [SuperOffice for Outlook][51] - it's available in six languages!
+* The user guide for [SuperOffice for Outlook][51] - it's available in six languages.
+* See the user guide for [SuperOffice Mobile CRM][52] in Docs.
+
+#### Developer portal
+
+* **Tenant manager** - separate pages to help manage and view customer (tenant) information, including authorizations and licenses, across different environments like development and production.
```
Comment (digitaldiina) [inline release-notes/10.3/index.md:119]: I suggest to use same names as in the portal, meaning SOD and not development

**Review by digitaldiina — CHANGES_REQUESTED** [review]
Please update the release date in the metadata header too.

**Review by kirstiaa — COMMENTED** [review]
Done changing meta data information

**Review by digitaldiina — APPROVED** [review]
✅

**Pull request merged**